_run/common.mk

```diff
@@ -39,6 +39,7 @@ SERVER_CERTS := provider
 
 
 .PHONY: init
+init: AKASH_GAS='' AKASH_GAS_PRICES=''
```

Comment:
`node-init-gentx` seems to be better place

`node-init-gentx: AKASH_GAS='' AKASH_GAS_PRICES=''`

Reply:
Why? There isn't any place during the `init` step where we should need to compute gas or fees.

Reply:
it's probably estimating gas by sending a simulation transaction

Reply:
fails [here](https://github.com/ovrclk/akash/blob/9671afe7f55748473cf3cb5230aea23f8727366a/_run/common.mk#L104)